app.py

```diff
@@ -1,23 +1,32 @@
 #!/usr/bin/env python3
-import random
-import string
+# Installed
+from aws_cdk import App, Environment
 
-import aws_cdk as cdk
+# Local
+from sds_data_manager.utils.stackbuilder import build_sds
 
-from sds_data_manager.sds_data_manager_stack import SdsDataManagerStack
+"""
+This app is designed to be the dev and production deployment app. It defaults to a dev deployment
+via a default `env` value in cdk.json. To deploy to prod, specify `--context env=prod`.
+"""
 
-app = cdk.App()
+app = App()
 
-SDS_ID = app.node.try_get_context("SDSID")
+# Grab values from context
+sds_region = app.node.try_get_context("sds-region")
+where_to_deploy = app.node.try_get_context("env")
```

Comment:
Might want to update this variable to match "env"... but also, it's fine if you'd prefer to leave it, I think it still makes sense